src/main/java/com/booksaw/betterTeams/integrations/LuckPermsManager.java

```diff
@@ -0,0 +1,98 @@
+package com.booksaw.betterTeams.integrations;
+
+import com.booksaw.betterTeams.PlayerRank;
+import com.booksaw.betterTeams.Team;
+import com.booksaw.betterTeams.TeamPlayer;
+import com.booksaw.betterTeams.integrations.placeholder.TeamPlaceholderOptionsEnum;
+import net.luckperms.api.LuckPerms;
+import net.luckperms.api.context.ContextCalculator;
+import net.luckperms.api.context.ContextConsumer;
+import net.luckperms.api.context.ContextSet;
+import net.luckperms.api.context.ImmutableContextSet;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+public class LuckPermsManager implements ContextCalculator<Player> {
+
+	private final LuckPerms luckPerms;
+
+	public LuckPermsManager(LuckPerms luckPerms) {
+		this.luckPerms = luckPerms;
+	}
+
+	@Override
+	public void calculate(@NotNull Player player, @NotNull ContextConsumer consumer) {
+		Team team = Team.getTeam(player);
+
+		consumer.accept("bt_inteam", String.valueOf(team != null));
+		if (team == null) {
+			return;
+		}
+		TeamPlayer teamPlayer = team.getTeamPlayer(player);
+
+		// Player specific
+		addContext(consumer, "bt_rank", TeamPlaceholderOptionsEnum.RANK.applyPlaceholderProvider(team, teamPlayer), true);
+		addContext(consumer, "bt_teamchat", TeamPlaceholderOptionsEnum.TEAMCHAT.applyPlaceholderProvider(team, teamPlayer));
+
+		// Team info
+		addContext(consumer, "bt_level", TeamPlaceholderOptionsEnum.LEVEL.applyPlaceholderProvider(team, teamPlayer));
+		addContext(consumer, "bt_open", TeamPlaceholderOptionsEnum.OPEN.applyPlaceholderProvider(team, teamPlayer));
+		addContext(consumer, "bt_pvp", TeamPlaceholderOptionsEnum.PVP.applyPlaceholderProvider(team, teamPlayer));
+		addContext(consumer, "bt_hashome",  String.valueOf(team.getTeamHome() != null));
```

Comment:
Why is the TeamPlaceholderOptionsEnum.HASHOME not being used to match the other values?